.php-cs-fixer.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Do we need both `.php-cs-fixer.php` and `.php_cs`? I can delete the latter and still run the linter successfully.

The docs for v2.19 only mention `php-cs-fixer.php`: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.19.3/doc/config.rst